src/middlewared/middlewared/alembic/versions/12.0/2020-01-14_13-26_add_service_announcments.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 39a133a04496
+Revises: 133f2d9049d2
+Create Date: 2020-01-14 13:26:02.169132+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '39a133a04496'
+down_revision = '133f2d9049d2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('network_globalconfiguration', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('gc_service_announcement', sa.JSON(type=dict), nullable=True))
```

Comment:
It won't work IIRC, it should be just `sa.TEXT()`

Reply:
Okay. Thanks.